pkg/loader/compose/compose.go

```diff
@@ -182,9 +182,10 @@ func (c *Compose) LoadFile(file string) kobject.KomposeObject {
 				logrus.Warningf("Unsupported key networks - ignoring")
 				networksWarningFound = true
 			}
-			kobject.CheckUnsupportedKey(composeServiceConfig)
+			kobject.CheckUnsupportedKey(composeServiceConfig, provider)
```

Comment:
I'm not sure if Loader should check for unsupported keys that are provider specific.

Maybe we need to add other check for unsupported options in Tranfromer code.

Loader checks for keys that can load. Check in Transformer is done on kobject and it checks if there are keys that it can't transform.

This is not going to be that trivial change, so it might be better to do it in separate PR. This is quite big PR already.